### ISSUE TITLE
Add 3 dots icon to menu icon in UI elements

### ISSUE THREAD

**Issue by imatwawana** [grafana/writers-toolkit#811]
Add an image of a 3 dots menu to the menu icon entry in the UI elements list

**Comment by heitortsergent** [comment]
This might be a separate issue, but I've noticed that the icon is something that Google includes in their style guide: https://developers.google.com/style/ui-elements. It would be nice if we could include the icon in our docs as well. 🤓
 
![Screenshot 2024-08-26 at 4 32 31 PM](https://github.com/user-attachments/assets/32ec10af-32b7-42b2-a067-08bd4eac6a3e)

**Comment by jdbaldry** [comment]
Asked in #grafana-frontend-product-design: https://raintank-corp.slack.com/archives/C043GQQN7T8/p1724742569092689

**Comment by jdbaldry** [comment]
https://github.com/grafana/grafana/tree/main/packages/grafana-icons/svg

**Comment by jdbaldry** [comment]
https://developers.grafana.com/ui/latest/index.html?path=/story/docs-overview-icon--icons-overview

**Comment by jdbaldry** [comment]
@Clarity-89 noted

> The grafana-icons package is not released yet and could be subject to change. All the icons that are currently used are here: https://github.com/grafana/grafana/tree/main/public/img/icons. Most of them are in the unicons folder.

**Comment by heitortsergent** [comment]
That's really neat, thanks for following up on this 🤓 . What's the best way to use those icons in our Markdown files? Would we have to use HTML? Or could we use some sort of shortcode?

**Comment by jdbaldry** [comment]
We probably want to use a shortcode I think. I probably can't put that together today but when I find some time, I can open a website PR and start the discussion with the team there.

**Comment by heitortsergent** [comment]
Nice, thanks @jdbaldry! And sorry for hijacking this issue. 😅

**Comment by imatwawana** [comment]
Before you get too far on the icons thing @jdbaldry, I think I'd like to put this up for discussion in the style roundtable. I'm not sure if following this rule would add a non-trivial amount of work for folks and since we're not doing it currently, I'd like to have a discussion before we start adding icon images to our docs. The only place I've seen folks do this so far is with the **Add** button which often has a `+` sign as part of the button text, but it's not actually an icon, it's just a button with slightly odd text. @heitortsergent, can you add this to the style roundtable agenda, please?

**Comment by jdbaldry** [comment]
I'm reopening this because I think the linked PR wasn't intended to close this issue.

**Comment by jdbaldry** [comment]
Oh yeah, my bad!

**Comment by heitortsergent** [comment]
Sorry I started the confusion here 😅 . I added this to our next roundtable discussion, thanks @imatwawana! 🙇
